package.json

```diff
@@ -89,9 +93,17 @@
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
-  "dependencies": {
-    "carbon-components": "^10.20.0",
+  "peerDependencies": {
+    "@openmrs/esm-api": "^3.0.0",
+    "@openmrs/esm-error-handling": "^1.1.1",
+    "carbon-components": "^10.19.0",
     "carbon-components-react": "^7.20.0",
```

Comment:
Right now this package is not shared. I am not sure if that will change. Ideally, leave it in `dependencies` (this is the reason why I did not mention this package, but only the "vanilla" carbon packages).